apps/cms-v2/README.md

```diff
@@ -0,0 +1,83 @@
+<div style="text-align: center">
+  <img width="150" alt="" src="./public/logo.png">
+</div>
+
+<div style="text-align: center">
+  <h1>Saleor App CMS</h1>
+
+  <p>CMS is a Saleor app that allows one-direction synchronization of Saleor products into supported CMS platforms.</p>
+</div>
+
+<div style="text-align: center">
+  <a target="_blank" rel="noopener noreferrer" href="https://docs.saleor.io/developer/app-store/apps/cms">Docs</a>
+<br><br>
+</div>
+
+### Here are the currently available platforms:
+
+- [Contentful](https://www.contentful.com/)
+- [Strapi](https://strapi.io/)
+- [DatoCMS](https://www.datocms.com/)
+- [Builder.io](https://www.builder.io/)
+- [Payload](https://payloadcms.com/)
+
+### How to use this project
+
+#### Requirements
+
+- [node v16](http://nodejs.com)
+- [pnpm](https://pnpm.io/)
+- [ngrok](https://ngrok.com/)
+- Saleor Cloud account (free!) or local instance
+- An account of one of cms providers listed above
+
+#### Running app in development mode
+
+1. Install the dependencies by running the following command in the shell:
+
+```shell
+pnpm install
+```
+
+2. Create a file named `.env` and use the contents of the [`.env.example`](./.env.example) file as a reference. Set you `SECRET_KEY` in [`.env`](./.env)
```

Comment:
`.env.example` says that SECRET_KEY is required only for production build

Reply:
`APP_API_BASE_URL` and `APP_IFRAME_BASE_URL` - sounds like something to configure